packages/spright-components/src/utilities/style/display.ts

```diff
@@ -0,0 +1,9 @@
+import {
+    type CSSDisplayPropertyValue,
+    // eslint-disable-next-line no-restricted-imports
+    display as foundationDisplay
+} from '@microsoft/fast-foundation';
+
+export const display: typeof foundationDisplay = (
```

Comment:
Where did we land on code sharing vs duplication of utilities between Nimble and Spright? I see we have a few other duplicate files in `src/utilities`. Are we tracking the work to share them at some point?

If we're ok duplicating for now, then let's add the same comment as suggested in the nimble-components copy.

Reply:
Don't think we have an agreed on policy quite yet. I don't really want nimble-components to become a component development infra / tools package + components package so right now we are duplicating utilities. I'll duplicate the comment for now 👍